### PR TITLE
Update alert links to v3 url structure [AO-12992] and remove beacon 😢

### PR DESCRIPTION
**Pull request by kangaechigai**
:ticket: https://swicloud.atlassian.net/browse/AO-12992

(replacing https://github.com/librato/librato-services/pull/168)

Now that we're transitioning to the v3 alerts UI in AO, we want to update notification links to use the new URL structure, including org ID. I _think_ `payload['user_id']` always exists now (despite it only being present in the `sample_new_alert_payload`) and actually matches the org ID (thanks to org ghost owner magic). I had to update the tests with that assumption, so hopefully it's a valid one.

This is a pretty naive first pass. I would greatly appreciate feedback and advice on next steps / how to actually get this out there, since I'm not at all familiar with how/where this code actually gets run.

### CODE REVIEW

**Comment by kangaechigai** [comment]
I apparently can't directly request reviewers, but @akahn @petercable can you please take a look?

File: services/beacon.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def receive_alert
         alert_instance_id: payload[:incident_key],
         alert_instance_origination_time: payload[:trigger_time],
         description: settings[:description],
-        url: alert_link(payload[:alert][:id])
+        url: alert_link(payload.with_indifferent_access)
```
Comment (akahn) [inline services/beacon.rb]: I think this file can be removed altogether.

Comment (kangaechigai) [reply services/beacon.rb]: heh I just searched for every `alert_link` and made the necessary changes, didn't even pay attention to the service name. I'll go ahead and remove all the beacon stuff.

Comment (dkuebric) [reply services/beacon.rb]: RIP ☠️

**Comment by kangaechigai** [comment]
I don't have access to merge this or to tag a new version. Anyone wanna help me out with that?

**Pull request merged**